internal/rbd/nodeserver.go

```diff
@@ -100,10 +104,9 @@ var (
 	// checking the support for reflink.
 	xfsHasReflink = xfsReflinkUnset
 
-	mkfsDefaultArgs = map[string][]string{
-		"ext4": {"-m0", "-Enodiscard,lazy_itable_init=1,lazy_journal_init=1"},
-		"xfs":  {"-K"},
-	}
+	// ext4HasPrezeroed is set by ext4SupportsPrezeroed(), use the function when
+	// checking the support for assume_storage_prezeroed.
+	ext4HasPrezeroedSupport = ext4PrezeroedUnset
```

Comment:
I understand you copied the `xfsHasReflink` idea. But I think it is cleaner if the `NodeServer` struct has a field for these kind of detectable parameters. Have you thought about doing it that way?